maat-court-data-api/src/main/java/gov/uk/courtdata/repository/ConcorContributionsRepository.java

```diff
@@ -0,0 +1,13 @@
+package gov.uk.courtdata.repository;
+
+import gov.uk.courtdata.dces.enums.ConcorContributionStatus;
+import gov.uk.courtdata.entity.ConcorContributionsEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import java.util.List;
+
+@Repository
+public interface ConcorContributionsRepository extends JpaRepository<ConcorContributionsEntity, Integer> {
+    List<ConcorContributionsEntity> findByStatus(ConcorContributionStatus status);
+    List<ConcorContributionsEntity> findByIdIn(List<String> ids);
```

Comment:
I think the method signature would communicate the intent of this method better by using a `Set` to hold the `ids` as I would imagine the result of calling `findByIdIn` in all the scenarios below would be identical?

```
- findByIdIn(Arrays.asList("1", "3"));
- findByIdIn(Arrays.asList("1", "3", "1"));
- findByIdIn(Arrays.asList("3", "1"));
- findByIdIn(Arrays.asList("3", "3", "1"));
```